pyproject.toml

```diff
@@ -14,6 +14,7 @@ fastapi = "^0.62.0"
 uvicorn = "^0.13.1"
 Jinja2 = "^2.11.2"
 aiofiles = "^0.6.0"
+# :index:`fixme`: shouldn't we install a `backend <https://www.encode.io/databases/#installation>`_?
```

Comment:
So far I've been working with sqllite.  But we should also install what we need for postgresql when we get to that.

Reply:
Should I therefore make this `pip install databases[sqlite]` for now?

Reply:
If databases[sqlite,postgresql] works that is maybe the best option?  As of now I have explicitly installed what we need for both, but perhaps leaving those as somewhat hidden dependencies is better than calling them out.

Reply:
It looks like `databases = {extras = ["sqlite", "postgresql"], version = "^0.4.1"}` works. I'll add that and remove the `aiosqlite` entry.